File: lua/refactoring/tests/refactor/123/ts/multiple-vars/inline_var.start.ts
type Order = {
    quantity: number;
    itemPrice: number;
}

// This the straight from the book (slight modifications)
function orderCalculation(order: Order) {
    const basePrice = order.quantity*order.itemPrice, i = 3;
    basePrice;
    return basePrice -
        Math.max(0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(basePrice * 0.1, 100);
}

File: lua/refactoring/tests/refactor/123/csharp/multiple-vars/inline_var.start.cs
namespace Test
{
    class Order
    {
        int quantity { get; set; }
        int itemPrice { get; set; }
    }

    class Inline
    {
        double orderCalculation(Order order)
        {
            float basePrice = order.quantity * order.itemPrice, i = 3.0f;
            return basePrice -
                Math.Max(0, order.quantity - 500) * order.itemPrice * 0.05 +
                Math.Min(basePrice * 0.1, 100.0);
        }
    }
}

